### ISSUE TITLE
Allow users to manupulate the slides they want to see.

### ISSUE THREAD

**Issue by Jy4058350** [Jy4058350/DevJourney#196]
1 .Stop slides
2. Stops at the slide you want to stop.
3. implementation of buttons to control slides.

**Comment by Jy4058350** [comment]
There is issue of textSlide slide call 2 next 8.

**Comment by Jy4058350** [comment]
Store:

When we say "store," we usually mean creating a reference or variable that points to a particular DOM element.
In your code, const circle = document.createElement("div"); creates a new <div> element and stores a reference to it in the variable circle.
Append:

"Append" refers to adding an element as a child to another element in the DOM.
The line circleContainer.appendChild(circle); appends the circle element (which is stored in the variable) as a child to the circleContainer element.
In summary, storing is about creating a reference to an element, and appending is about adding an element to another element as a child. In the context of your code, you are both creating a reference (circle variable) and appending the newly created <div> element to the circleContainer.

**Comment by Jy4058350** [comment]
complete
